docs/source/science/HLS/HLS-STAC-geoparquet.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<p>Can we split this into an "Importing Packages" and "Searching the Data Store" sections?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/538/discussion/#comment-2624175976'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>